packages/common/src/ui/components/inputs/TextInput/BasicTextInput.tsx

```diff
@@ -21,6 +21,8 @@ export const BasicTextInput: FC<StandardTextFieldProps> = React.forwardRef(
         disableUnderline: true,
         ...InputProps,
         sx: {
+          border: theme =>
+            error ? `2px solid ${theme.palette.error.main}` : 'none',
```

Comment:
I quite like this. Needs a nice 'invalid' message also though, I think.

Reply:
works really well with the date picker!